examples/next/package.json

```diff
@@ -3,7 +3,7 @@
   "private": true,
   "version": "0.5.8",
   "scripts": {
-    "dev": "next dev -p 3002",
+    "dev": "next dev -p 3002 --experimental-https",
```

Comment:
Turning the example to HTTPS so we can actually test the prod directly and avoid this:
![image](https://github.com/user-attachments/assets/6ea977ae-e06d-4c7b-8fee-a5d1b42f975f)